app/models/user.rb

```diff
@@ -35,6 +35,7 @@ class User < ActiveRecord::Base
          :password_archivable, # in signonotron2/lib/devise/models/password_archivable.rb
          :password_expirable   # in signonotron2/lib/devise/models/password_expirable.rb
 
+  validates :require_2sv, inclusion: [false], if: ->(u) { u.otp_secret_key.present? }
```

Comment:
This seems to ensure that `require_2sv` is set to false once 2SV has been set up for a user, but I don't see any code that sets this once 2SV is set up. This will also result in a confusing error message for admins if they try to turn this flag on for a user that already has 2SV set up.